From 9ce76aeebf31018b884a6425277dd4c9fb5ea336 Mon Sep 17 00:00:00 2001
From: Demetris Ierokipides <ierokipides.dem@gmail.com>
Date: Wed, 30 Dec 2020 03:55:06 +0000
Subject: [PATCH 2/2] ARM: dts: rockchip: add extra cpu opp points to
 rk3288-miqi

Add extra 1.7GHz and 1.8GHz opp points to the MiQi device-tree to improve
performance.

Signed-off-by: Demetris Ierokipides <ierokipides.dem@gmail.com>
---
 arch/arm/boot/dts/rk3288-miqi.dts | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288-miqi.dts b/arch/arm/boot/dts/rk3288-miqi.dts
index 713f55e143c6..1b48855db6ff 100644
--- a/arch/arm/boot/dts/rk3288-miqi.dts
+++ b/arch/arm/boot/dts/rk3288-miqi.dts
@@ -96,6 +96,18 @@ &cpu3 {
 	cpu-supply = <&vdd_cpu>;
 };
 
+&cpu_opp_table {
+	opp-1704000000 {
+		opp-hz = /bits/ 64 <1704000000>;
+		opp-microvolt = <1350000>;
+	};
+
+	opp-1800000000 {
+		opp-hz = /bits/ 64 <1800000000>;
+		opp-microvolt = <1350000>;
+	};
+};
+
 &emmc {
 	bus-width = <8>;
 	cap-mmc-highspeed;
-- 
2.25.1

